src/content/docs/infrastructure/infrastructure-troubleshooting/troubleshoot-logs/infrastructure-agent-logging-behavior.mdx

```diff
@@ -89,11 +89,11 @@ Agent handles each STDERR line as follows:
 * **when agent runs in verbose mode**: it just forwards the full STDERR line as a DEBUG agent log entry placing integration line contexts within the \`msg\` field.
 * **otherwise**: it parses the line against the expected format (see below) and only logs as agent ERROR level, entries produced by integrations with \`fatal\` or \`error\` severity levels. In this case fields are extracted and forwarded in structured manner (therefore if JSON output is enabled for the agent fields become queryable.
 
-The infrastructure agent is also capable of filtering the errors of an integration if it's explicitly defined in the log configuration. In this example the errors coming from nri-mssql won't be shown when the agent log is in any level except debug:
+The infrastructure agent is filtering by default all the errors of an integration that didn't cause it to exit. All the errors of an integration will only be shown if the log level is DEBUG or if the integration is explicitly included in the log configuration. In this example all the errors coming from nri-mssql will be shown even if the log level is INFO:
```

Comment:
```suggestion
By default, the infrastructure agent filters out any errors from integrations that don't prevent the integration from running. You'll only see all errors from an integration if the log level is set to DEBUG, or if the integration is specifically listed in the log configuration. For example, in this configuration, all errors coming from `nri-mssql` will be shown, even if the log level is INFO:
```

@alvarocabanas the phrase "the errors of an integration that didn't cause it to exit" is a bit confusing. Does my revision above make sense?